vexim/config/tests/functionsValidatePasswordTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+	
+class FunctionValidatePasswordTest extends PHPUnit_Framework_TestCase {
+		
+	public function setUp() {
+		require_once(__DIR__ . '/../functions.php');
+	}
+
+        public function testSendEmptyPasswords()
+        {
+            $this->assertFalse(validate_password("", ""));
+        }
+
+        public function testSendSamePasswords()
+        {
+            $this->assertTrue(validate_password("password", "password"));
+        }
+
+        public function testSendDifferentPasswords()
+        {
+            $this->assertFalse(validate_password("password", "pass"));
+        }
+
+        public function testSendNullValues()
+        {
+            $this->assertFalse(validate_password(null, null));
+        }
+
+        public function testSendBoolValues()
+        {
+            $this->assertFalse(validate_password(true, true));
+            $this->assertFalse(validate_password(false, false));
+        }
+
+        public function testSendIntegerValues()
+        {
+            $this->assertFalse(validate_password(10, 10));
+        }
+
+        public function testSendZeroValues()
+        {
+            $this->assertFalse(validate_password(0, 0));
+        }
+}
+
+### EOF
```

Comment:
This is to check and prove the functionality of the code https://en.wikipedia.org/wiki/Unit_testing 
It helps keep code working when you make changes. 
Next time when a change is made to validate_password() this file can be run with phpunit to make sure the function still works with the existing code.